src/Common/src/TypeSystem/Common/TargetDetails.cs

```diff
@@ -33,6 +33,22 @@ public enum TargetOS
     }
 
     /// <summary>
+    /// Specifies 
+    /// </summary>
+    public enum TargetApplicationModel
```

Comment:
@jkotas Would `TargetAbi` work? I had objections to the term "Runtime" because Runtime is the thing that is pretty much the same between N and RT.